app/assets/javascripts/countries/views/show/CountryModalView.js

```diff
@@ -92,6 +103,8 @@ define([
 
     _cleanIndicators: function() {
       $('.indicators-group li').removeClass('is-selected');
+      $('#jurisdictions-list li').removeClass('is-selected');
```

Comment:
Uhm.. Do you mean a function that saves the reference of these elements and use them instead of the jQuery selector?